### ISSUE TITLE
Installer v0.2.1: Error: spawn path-to-code.cmd ENOENT

### ISSUE THREAD

**Issue by Hi-jin2204** [microsoft/vscode-java-pack#285]
installer version: 0.2.1
env:
```
os: win32
arch: x64
PATH: C:\Program Files (x86)\Google\Chrome\Application;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;D:\java\jdk1.8.0_121\bin;C:\Users\장혜진\AppData\Local\Microsoft\WindowsApps;
JAVA_HOME: D:\java\jdk1.8.0_121
JDK_HOME: undefined
```
log:
```
[오후 10:42:06] Configure Java Settings
[오후 10:42:07] Configure Java Settings ... [Completed]
[오후 10:42:07] Install Java Extensions
[오후 10:42:07] Error occurred
[오후 10:42:07] Error: spawn C:\Users\장혜진\AppData\Local\Programs\Microsoft VS Code\bin\code.cmd ENOENT

```

**Comment by Eskibear** [comment]
execa@1.0.0 was used to execute corresponding commands, and I notice they've fixed a bunch of bugs since 2.x. 

I'll upgrade it to the latest version, see if the issue still exists.

**Comment by Hi-jin2204** [comment]
what an earth is this??!!!! i've never use github except download the file to learn code .... jeeze

**Comment by Hi-jin2204** [comment]
oh no.. what should i do.......... that's why all of sudden my computer went slow down

**Comment by Eskibear** [comment]
we just shipped a new version v0.2.2 to fix this.

**Comment by Hi-jin2204** [comment]
Thank you! anw i formatted my laptop was too slow to do things :)
